litellm/litellm_core_utils/litellm_logging.py

```diff
@@ -433,6 +435,22 @@ def pre_call(self, input, api_key, model=None, additional_args={}):
                             message=f"Model Call Details pre-call: {details_to_log}",
                             level="info",
                         )
+                    elif callback == "literalai":
```

Comment:
if the LiteralAILogger inherits from CustomLogger (or CustomBatchLogger) this should not be necessary. 

All you need to do is add it here - https://github.com/BerriAI/litellm/blob/ab153c3e8dab6be093eba3726ada19da176ee8cc/litellm/litellm_core_utils/litellm_logging.py#L2137